undertow/src/main/java/org/wildfly/elytron/web/undertow/server/ElytronRunAsHandler.java

```diff
@@ -55,11 +53,11 @@ public ElytronRunAsHandler(HttpHandler next, BiFunction<SecurityIdentity, HttpSe
     @Override
     public void handleRequest(HttpServerExchange exchange) throws Exception {
         SecurityContextImpl securityContext = (SecurityContextImpl) exchange.getSecurityContext();
-        Account account = securityContext != null ? securityContext.getAuthenticatedAccount() : null;
-        SecurityIdentity securityIdentity = (account instanceof ElytronAccount) ? ((ElytronAccount)account).getSecurityIdentity() : null;
+        SecurityIdentity securityIdentity = securityContext != null ? securityContext.getSecurityIdentity() : null;
```

Comment:
Need to think about this a little more, the serializability of the account really affected clustering scenarios.  Need to double check in what other scenarios this ElytronAccount is re-used for a subsequent request, in a subsequent request we know the SecurityContext will be new so the SecurityIdentity will be null but do we have cases where ElytronAccount would not be null in a subsequent request?

Reply:
In theory, should not be ElytronAccount stored when FORM/SPNEGO re-auth is used?
But I am not able to find it in code... :/

Reply:
But it is useless if we need securityIdentity for something....

Reply:
In any case, `-DallTests` is passing